sigar_linux_common.go

```diff
@@ -55,20 +55,26 @@ func (self *LoadAverage) Get() error {
 func (self *Mem) Get() error {
 	var buffers, cached uint64
 	table := map[string]*uint64{
-		"MemTotal": &self.Total,
-		"MemFree":  &self.Free,
-		"Buffers":  &buffers,
-		"Cached":   &cached,
+		"MemTotal":     &self.Total,
+		"MemFree":      &self.Free,
+		"MemAvailable": &self.ActualFree,
+		"Buffers":      &buffers,
+		"Cached":       &cached,
 	}
 
 	if err := parseMeminfo(table); err != nil {
 		return err
 	}
 
-	self.Used = self.Total - self.Free
 	kern := buffers + cached
-	self.ActualFree = self.Free + kern
-	self.ActualUsed = self.Used - kern
+	if self.ActualFree > 0.0 {
```

Comment:
That's a good point. I would need to refactor the code to be able to distinguish between `MemAvailable: 0kB` and `MemAvailable` not present.